services/recipe.service.js

```diff
@@ -0,0 +1,21 @@
+//should be from database
+const {recipes, ROLE} = require('../data')
+
+exports.findAllRecipes = async () => {
+    return recipes;
+}
+
+exports.findUserRecipes = async (user) => {
+    return recipes.filter(recipe => recipe.username === user.name);
```

Comment:
just leave empty function. I think collections can have different methods we can actually use them with mongo filtration